ivadomed/object_detection/utils.py

```diff
@@ -0,0 +1,268 @@
+import json
+import os
+import statistics
+
+import nibabel as nib
+import numpy as np
+from scipy import ndimage
+
+from ivadomed import postprocessing as imed_postpro
+from ivadomed import transforms as imed_transforms
+from ivadomed import utils as imed_utils
+from ivadomed.loader import utils as imed_loader_utils
+
+
+def get_bounding_boxes(mask):
+    """
+    Generates a 3D bounding box around a given mask.
+    :param mask: numpy array with the mask of the ROI
```

Comment:
can you please use the [Google style docstrings](https://github.com/neuropoly/ivadomed/blob/master/CONTRIBUTING.md#documentation-and-docstrings)? thx